### PR TITLE
Made MultiQC protocol, for all qc files.

### PR DESCRIPTION
**Pull request by BenjaminsM**

### CODE REVIEW

File: protocols/MultiQC.sh [hunk 1]
```diff
@@ -2,61 +2,133 @@
...
+        - 30
+        - 40
+        - 50
+        - 100" >> ${intermediateDir}/${project}.multiqc_config.yaml
+
+
+module load multiqc/1.0-foss-2015b-Python-2.7.11
+module list
+
+multiqc -c ${intermediateDir}/${project}.multiqc_config.yaml -f ${intermediateDir} -o ${intermediateDir}
```
Comment (marieke-bijlsma) [inline protocols/MultiQC.sh]: -f = -l

Comment (BenjaminsM) [reply protocols/MultiQC.sh]: -f == overwrite report, ${intermediateDir} has nothing to do with the -f option, its just the input folder in where multiQC is looking for the QC files.

File: protocols/MultiQC.sh [hunk 2]
```diff
@@ -2,61 +2,133 @@
...
-#string caddVersion
+#string hpoVersion
+
+set -e
+set -u
+
+#Making Header of the MultiQC Report
+echo -e "report_header_info:
+    - Contact E-mail: 'helpdesk.gcc.groningen@gmail.com'
+    - Pipeline Version: '${ngsversion}'
```
[the remark this answers is not in the document]
Comment (BenjaminsM) [reply protocols/MultiQC.sh]: ngsversion=$(module list | grep -o -P 'NGS_DNA(.+)');\

Defined in generate_template.csv and now also as #string ngsversion in the script

File: protocols/MultiQC.sh [hunk 3]
```diff
@@ -2,61 +2,133 @@
...
+    - Project : '${project}'
+    - capturingKit : '${capturingKit}'
```
[the remark this answers is not in the document]
Comment (BenjaminsM) [reply protocols/MultiQC.sh]: defined in the samplesheet.csv and also as #string capturingkit in the script

File: protocols/MultiQC.sh [hunk 4]
```diff
@@ -2,61 +2,133 @@
...
+        - 10
+        - 20
+        - 30
+        - 40
+        - 50
+        - 100" >> ${intermediateDir}/${project}.multiqc_config.yaml
+
+
+module load multiqc/1.0-foss-2015b-Python-2.7.11
+module list
```
[the remark this answers is not in the document]
Comment (BenjaminsM) [reply protocols/MultiQC.sh]: done

File: protocols/MultiQC.sh [hunk 5]
```diff
@@ -2,61 +2,133 @@
...
+        - 2
+        - 10
+        - 20
+        - 30
+        - 40
+        - 50
+        - 100" >> ${intermediateDir}/${project}.multiqc_config.yaml
+
+
+module load multiqc/1.0-foss-2015b-Python-2.7.11
```
[the remark this answers is not in the document]
Comment (BenjaminsM) [reply protocols/MultiQC.sh]: done :-)

File: protocols/MultiQC.sh [hunk 6]
```diff
@@ -43,6 +43,13 @@
 #string xhmmVersion
 #string hpoVersion
 
+#string ngsversion
+#string project
```
Comment (marieke-bijlsma) [inline protocols/MultiQC.sh]: Duplicate

Comment (BenjaminsM) [reply protocols/MultiQC.sh]: removed

**Pull request merged**